nowcasting_dataset/data_sources/pv/live.py

```diff
@@ -120,7 +125,12 @@ def get_pv_power_from_database(
     pv_yields_df = empty_df.join(pv_yields_df)
     limit = int(interpolate_minutes / 5)
     if limit > 0:
-        pv_yields_df.interpolate(limit=limit, inplace=True)
+        try:
+            pv_yields_df.interpolate(limit=limit, inplace=True, method="cubic")
+        except Exception as e:
+            logger.exception(e)
+            logger.debug(pv_yields_df)
```

Comment:
also, I think this might fail, because `logger.debug` expects a string not a df?

Reply:
no this is fine, it prints out the same as if you did print(df)